install/k8s/contiv/contiv_devtest.yaml

```diff
@@ -0,0 +1,275 @@
+---
+# This ConfigMap is used to configure a self-hosted Contiv installation.
+# It can be used with an external cluster store(etcd or consul) or used
+# with the etcd instance being installed as contiv-etcd
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: contiv-config
+  namespace: kube-system
+data:
+  # The location of your cluster store. This is set to the
+  # avdertise-client value below from the contiv-etcd service.
+  # Change it to an external etcd/consul instance if required.
+  cluster_store: "etcd://__NETMASTER_IP__:6666"
+  # The CNI network configuration to install on each node.
+  cni_config: |-
+    {
+      "cniVersion": "0.1.0",
+      "name": "contiv-net",
+      "type": "contivk8s"
+    }
+  config: |-
+    {
+       "K8S_API_SERVER": "https://__NETMASTER_IP__:6443",
+       "K8S_CA": "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt",
+       "K8S_KEY": "",
+       "K8S_CERT": "",
+       "K8S_TOKEN": ""
+    }
+---
+
+# This manifest installs the Contiv etcd on the kubeadm master. 
+# If using an external etcd instance, this can be deleted. This uses a DaemonSet
+# to force it to run on the master even when the master isn't schedulable, and uses
+# nodeSelector to ensure it only runs on the master.
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: contiv-etcd
```

Comment:
Having to install at test-cluster bringup time is only going to increase the testing time. I would rather have the compatible versions of dependencies installed and test the functionality of netplugin here. The end-to-end testing should ideally be the place to test the install and netplugin together. 

Yes, I do have a parallel effort to get this in sync with k8s1.6 which install has and this is just the first pass to get things going.